src/components/Databoard-review.js

```diff
@@ -0,0 +1,133 @@
+        let liveData = matchData.liveData;
+        let lineupData = liveData.lineups;
+        let homeData = lineupData.home.stats;
+        let awayData = lineupData.away.stats;
+
+        setHomePossessionData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "possessionPercentage";
+            })[0][timekey]
+          )
+        );
+        setAwayPossessionData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "possessionPercentage";
+            })[0][timekey]
+          )
+        );
+
+        setHomeShotData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "totalScoringAtt";
+            })[0][timekey]
+          )
+        );
+        setAwayShotData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "totalScoringAtt";
+            })[0][timekey]
+          )
+        );
+
+        setHomeShotOnTargetData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "ontargetScoringAtt";
+            })[0][timekey]
+          )
+        );
+        setAwayShotOnTargetData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "ontargetScoringAtt";
+            })[0][timekey]
+          )
+        );
+
+        setHomeCornersData(
+          parseFloat(
+            homeData.filter((d) => {
+              return d.type === "wonCorners";
+            })[0][timekey],
+            0
+          )
+        );
+        setAwayCornersData(
+          parseFloat(
+            awayData.filter((d) => {
+              return d.type === "wonCorners";
+            })[0][timekey],
+            0
+          )
+        );
+        setDataLoaded(true);
+      });
+  }, [timekey]);
+  return dataLoaded ? (
+    <div className="board-display">
+      <ResultSegment
+        type="Possession"
+        homeMax={homePossessionData + awayPossessionData}
+        homeValue={homePossessionData}
+        awayMax={homePossessionData + awayPossessionData}
+        awayValue={awayPossessionData}
+      ></ResultSegment>
+
+      <ResultSegment
+        type="Shots"
+        homeMax={homeShotData + awayShotData}
+        homeValue={homeShotData}
+        awayMax={homeShotData + awayShotData}
+        awayValue={awayShotData}
+      ></ResultSegment>
+
+      <ResultSegment
+        type="Shots on Target"
+        homeMax={homeShotOnTargetData + awayShotOnTargetData}
+        homeValue={homeShotOnTargetData}
+        awayMax={homeShotOnTargetData + awayShotOnTargetData}
+        awayValue={awayShotOnTargetData}
+      ></ResultSegment>
+
+      <ResultSegment
+        type="Corners"
+        homeMax={homeCornersData + awayCornersData}
+        homeValue={homeCornersData}
+        awayMax={homeCornersData + awayCornersData}
+        awayValue={awayCornersData}
+      ></ResultSegment>
```

Comment:
I don't know what the data looks like that you're getting back from the API, but I would try and map through the data if possible, for example
```
  matchData.map((data)=> (
      <ResultSegment type={data.type} homeData={data.home.stats} awayData={data.away.stats}/>
    ))
```